packages/babel-plugin-transform-function-name/test/fixtures/issues/7199/output.js

```diff
@@ -0,0 +1,3 @@
+const x = {
+  [`y`]: function () {}
```

Comment:
actually i think we could evaluate simple expressions such as this to know the inferred function name